sourcecode.md

```diff
@@ -1,36 +1,22 @@
 #SONiC Source Repositories
```

Comment:
Looks good,  Can you also add the PAS and NAS repositories?  Because there are so many, perhaps just list a key repo with documentation for each?